diff --git a/components/PressCategory.vue b/components/PressCategory.vue
index b2ffedd2e205ffca4545bb355735463114d95088..b30ce35243b0eb6d727aa35d464a811eaec194f3 100644
--- a/components/PressCategory.vue
+++ b/components/PressCategory.vue
@@ -58,7 +58,15 @@ const { $tO } = useValaxyI18n()
         </RouterLink>
       </template>
 
-      <PressCategory v-else :category="categoryItem" :display-category="displayCategory" :collapsable="collapsable" />
+      <PressCategory 
+        v-else 
+        :category="categoryItem" 
+        :display-category="displayCategory" 
+        :collapsable="(() => {
+          const allChildrenAreNotCategoryList = Array.from(categoryItem.children.values()).every(child => !isCategoryList(child))
+          return allChildrenAreNotCategoryList ? categoryItem.total > 8 : false
+        })()" 
+      />
     </li>
   </ul>
 </template>
