diff --git a/client/composables/categories.ts b/client/composables/categories.ts
index 47642a6f17c4dff71dca9306a669bfd748cd197a..900ecd0ef88cffb1b45ee5de4949a1054e506aa6 100644
--- a/client/composables/categories.ts
+++ b/client/composables/categories.ts
@@ -67,6 +67,11 @@ export function useCategories(category?: MaybeRef<string>, posts: Post[] = []) {
       posts = site.postList
     }
 
+    posts.sort((a, b) => {
+      // console.log(a.top, b.top, 'sort post list')
+      return (b.top || 0) - (a.top || 0)
+    })
+
     const categoryList: CategoryList = {
       name: 'All',
       total: posts.length,
